src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -554,11 +557,34 @@ impl<T: AstInfo> AstDisplay for CreateConnectionStatement<T> {
         }
         f.write_node(&self.name);
         f.write_str(" TO ");
-        self.connection.fmt(f)
+        self.connection.fmt(f);
+        if let Some(validate) = self.validate {
```

Comment:
I am very leery of pretending to use the `WITH` options structure without actually using the same machinery as everywhere else. I realize it's a little annoying  for a single option, but believe the consistency and hygiene is worth the extra bit of effort.

Reply:
I went through the same dilemma and convinced myself that it's ok but since it smells to you too I'll do it the proper way